internal/server/auth/middleware.go

```diff
@@ -69,14 +105,42 @@ func UnaryInterceptor(logger *zap.Logger, authenticator Authenticator) grpc.Unar
 			return ctx, errUnauthenticated
 		}
 
-		if auth.ExpiresAt != nil && auth.ExpiresAt.AsTime().Before(time.Now()) {
```

Comment:
did this get moved somewhere else?

Reply:
I need a decent squint emoji on here. No I suspect this is a regression after a bad cherry pick. Good spot!